css-writing-modes-4/Overview.bs

```diff
@@ -2537,7 +2537,7 @@ Horizontal-in-Vertical Composition: the 'text-combine-upright' property</h3>
   Name: text-combine-upright
   Value: none | all | [ digits &lt;integer&gt;? ]
   Initial: none
-  Applies to: non-replaced inline elements
+  Applies to: [=inline boxes=] and text
```

Comment:
`text-combine-upright: all` is defined with

> Attempt to typeset horizontally all consecutive typographic character units within the box

So it's not clear what happens for text. Maybe "within the box or text run"?

Reply:
Hmm... this is an interesting point. I was originally thinking that applying this to text would only make sense for the `digits` value (which is why I didn't edit css-writing-modes-3) and not the `all` value. Can make that clearer, or can say that it only applies to boxes, or can apply it to text runs as you say. I'm not sure there's a real use case for the last one, so I think we should go with whatever makes the most sense to implementers. CC @litherum @kojiishi @jfkthame  What do you think? The situation is `span { display: contents; text-combine-upright: <something>; }`: should this have an effect or have no effect? If so should it work for both `all` and `digits` or only for one of them? It doesn't seem particularly useful, so the question is what's easiest to implement. :)

Reply:
```html
<div style="writing-mode: vertical-lr">
  <span style="display: contents; text-combine-upright: all">foo</span>
```
has effect in Firefox and Chromium. They don't support `digits`.

Reply:
OK, made the fixes in https://github.com/w3c/csswg-drafts/commit/0b31fa93ebe3630000c86f63c5f0dfe0da05280a

Reply:
Seems WebKit has `-webkit-text-combine: horizontal` instead of `text-combine-upright: all`, and it has no effect with `display: contents`. But it's a non-inherited property, so it's expected that the text doesn't inherit it, I guess.